opt_maven_install.sh

```diff
@@ -27,14 +26,15 @@ if [[ "$1" == "true" ]]; then
   ln -s /usr/bin/cmake3 /usr/bin/cmake
   export CMAKE_C_COMPILER=gcc CMAKE_CXX_COMPILER=g++
 
-  # build hadoop
+  # Build hadoop
   cd /build && mvn -B -e -Dtest=false -DskipTests -Dmaven.javadoc.skip=true clean package -Pdist,native -Dtar
   # Install prometheus-jmx agent
-  mvn dependency:get -Dartifact=io.prometheus.jmx:jmx_prometheus_javaagent:0.3.1:jar -Ddest=/build/jmx_prometheus_javaagent.jar
+  mvn dependency:get -Dartifact=io.prometheus.jmx:jmx_prometheus_javaagent:0.3.1:jar -Ddest=/build/jmx_prometheus_javaagent.jar && mv $HOME/.m2/repository/io/prometheus/jmx/jmx_prometheus_javaagent/0.3.1/jmx_prometheus_javaagent-0.3.1.jar /build/jmx_prometheus_javaagent.jar
```

Comment:
After playing around with this for the past couple of days, it appears the problem was that maven was storing this jar in the default .m2 repository location, despite the `-Ddest` flag being specified. I wasn't able to find much information on that flag, but it appears to me as someone that's maven-technically-challenged that flag value wasn't being respected. This is just a workaround for now, moving that jar we grab into the correct destination path, and we can further clean this up down the line.